parameter_files/WebParameters_2dii.yml

```diff
@@ -1,6 +1,6 @@
 default:
     paths:
-        # Prefer "../PACTA_analysis/" -- not "./" nor "/path/to/PACTA_analysis"
+       # Prefer "../this_repo/" -- not "./" nor "/path/to/this_repo"
```

Comment:
Using "PACTA_analysis" in the comment was a bad idea because causes an error 
in the integration-test.Rmd. Maybe I should do something about that -- but I don't 
want to over engineer this issue. I'll see if this bites again. For now I leave it at that.